Makefile

```
@@ -1,3 +1,9 @@
 tests:
 
 .PHONY: tests
+
+build:
+	docker build -t laysan-albatross-sexual-dimorphism .
+
+run:
+	docker run --name laysan-albatross-sexual-dimorphism-container -d -p 8888:80 laysan-albatross-sexual-dimorphism

```

Por favor por la versión larga de la instrucción:
```suggestion
	docker run --name laysan-albatross-sexual-dimorphism-container --detach --publish 8888:80 laysan-albatross-sexual-dimorphism

```

---

Va, lo cambio, pero espera, aún no revises esto, no está listo, por eso sigue con el [WIP] (Work in progress)

---

disculpa Efrén, andaba de mitotero solamente.